server/lobbyconnection.py

```diff
@@ -937,10 +937,14 @@ def launch_game(self, game, port, is_host=False):
         self.player.game = game
         self.player.game_port = port
 
+        args = ["/numgames " + str(self.player.numGames)]
```

Comment:
The reason I made it server-side is to that it's client-independent. But it's ok for me to have it client side....